app/boards/shields/ergos1_ose/ergos1_ose.zmk.yml

```diff
@@ -0,0 +1,12 @@
+file_format: "1"
```

Comment:
Looks like this file was copied from redox and not modified.

Reply:
You're right, I'll update that